internal/node/telemetry.go

```diff
@@ -140,14 +141,15 @@ func (t *Telemetry) initMeterProvider() error {
 				semconv.SchemaURL,
 				semconv.ServiceName(t.serviceName),
 				semconv.ServiceVersion(VERSION),
+				attribute.String("node_id", "1234"), // FIXME-- this should be a unique identifier for the node
```

Comment:
@kthomas I need access to the node public key...how would be the best way to get that here?